### ISSUE TITLE
Generate an implementation report

### ISSUE THREAD

**Issue by marcoscaceres** [w3c/webpayments-method-identifiers#54]
The spec is currently missing an implementation report. I've asked @yatri1609 to give us a hand with this! 👋

What we need to do:

- [x] copy/paste the [test file](https://w3c.github.io/payment-method-id/#comparison) referenced in the spec into: https://github.com/web-platform-tests/wpt/tree/master/payment-method-id. That way we have an independent test suite. 

- [x] Verify that everything in the spec is being tested (I believe it is, but want one more check). This involves looking at the source of the [test file](https://w3c-test.org/payment-request/payment-request-ctor-pmi-handling.https.html) - and making sure we've checked everything according to the rules of the spec.    

- [x]  Run the tests in at least 2 browsers (e.g., Chrome and Firefox) and generate the report by [following the instructions](https://github.com/w3c/webpayments/wiki/implementation-report-generation). However, instead of running against the "payment-request/" directory, it would be run against "/payment-method-id/".

- [x] Finally, we need to just update the "test-suite" link at the top of the spec to point to the new test suite location. So, we need to change this line: https://github.com/w3c/payment-method-id/blob/gh-pages/index.html#L38

**Comment by yatri1609** [comment]
Sure @marcoscaceres I'll do it

**Comment by ianbjacobs** [comment]
Thanks for the help!!

**Comment by yatri1609** [comment]
@marcoscaceres While running the test in the browsers using "payment-request/" directory, it made a json file while running under "payment-method-id/" directory it didn't made any json file.  So is it what is expected or something unusal?

**Comment by marcoscaceres** [comment]
So, just to confirm, did you copy/paste the file “payment-request/payment-request-ctor-pmi-handling.https.html” file into to the “payment-method-id/” directory? That needs to be in place for the test to run and produce the JSON. 

once you move that file, please send a pull request to the web platform test repo... once that lands, the JSON file will get produced when you run the test suite.

**Comment by yatri1609** [comment]
@marcoscaceres Ohk!

**Comment by yatri1609** [comment]
@marcoscaceres Yes I did copy/paste it into the payment-method-id directory. Now I need to send a PR to which repo as I'm getting confused with the repos. The changes are made in the wpt repo and the PR must also be send in the same right? Then I'll be testing and generating the JSON file which will go in the test-results repo. Am I going in the right direction?

**Comment by marcoscaceres** [comment]
> The changes are made in the wpt repo and the PR must also be send in the same right? 

Correct. Apologies for not being clear. 

> Then I'll be testing and generating the JSON file which will go in the test-results repo. Am I going in the right direction?

Yep 👍 that sounds correct.

**Comment by yatri1609** [comment]
Cool I'll send a PR to the wpt repo !

**Comment by marcoscaceres** [comment]
Ok, first item at the top off the list crossed off! ✨

**Comment by marcoscaceres** [comment]
Oh, to test in Firefox, make sure you do this:

1. Type “about:config” in the url bar and hit enter. 
1. click yes to “void my warranty” :)
1. Search for “dom.payments.request.enabled” and set it to true. 
1. Open a new tab and everything should be working. You might want to restart Firefox, just to be sure.

**Comment by yatri1609** [comment]
@marcoscaceres I did that and updated my PR but the file hasn't changed at all it was the same

**Comment by marcoscaceres** [comment]
I’ll take a look, but unfortunately it won’t be until a bit later today.

**Comment by yatri1609** [comment]
@marcoscaceres Yeah no problem! I'll also have a look tomorrow morning as it's already 4 in the morning and I have lectures tomorrow so I'll complete the other tasks tomorrow. :grinning:

**Comment by marcoscaceres** [comment]
Heh, was just thinking it must be super late for you :) I just tried it nonetheless, and the test is showing up. But yes, let’s finish this up tomorrow!

**Comment by marcoscaceres** [comment]
Spotted a typo above, it should have said to run against "/payment-method-id/" (was missing first “/“)

**Comment by yatri1609** [comment]
@marcoscaceres We need to update the link to which link exactly... Is it the link to the payment-method-id till I understood?

**Comment by marcoscaceres** [comment]
@yatri1609, so, in this repository, you just need to change:
https://github.com/w3c/payment-method-id/blob/gh-pages/index.html#L38

To:
```JS
testSuiteURI: "https://w3c-test.org/payment-method-id/",
implementationReportURI: "https://w3c.github.io/test-results/payment-method-id/all.html",
```

**Comment by yatri1609** [comment]
@marcoscaceres I have already done that.

**Comment by marcoscaceres** [comment]
@ianbjacobs, thanks to @yatri1609. we now have an implementation report showing 3 fully conforming implementations 🎉 

I'll create a new bug to track moving this to Proposed Recommendation.

**Comment by ianbjacobs** [comment]
@yatri1609 and @marcoscaceres,

Great work, thank you both very much!
🎉🎉🎉

**Comment by marcoscaceres** [comment]
closing as done via #58
